packages/react-native-reanimated/src/layoutReanimation/defaultTransitions/FadingTransition.ts

```diff
@@ -29,7 +29,7 @@ export class FadingTransition
     const delayFunction = this.getDelayFunction();
     const callback = this.callbackV;
     const delay = this.getDelay();
-    const duration = (this.durationV ?? 500) / 2;
+    const halfDuration = (this.durationV ?? 500) / 2; // The minimal duration is 100ms, otherwise the animation will be halfDuration + 50ms long
```

Comment:
Let't wait with changes in this particular file until the https://github.com/software-mansion/react-native-reanimated/pull/6183 will be merged.